### PR TITLE
Add support for `pypy` and `pypy3` interpreter and remove support for python 3.3

### LINKED ISSUE #9: Exception logging not respecting logzio_type
Unlike `logging.info(...)`, `logging.exception(...)` does not carry the logzio_type making it impossible to find these messages. Instead it overwrites type to:```type: logzio-invalid-log```.

### LINKED ISSUE #32: Add support for pypy2 and pypy3
When using this library in `pypy3` the logging is not being sent to LogzIO and there is no warning.

### PR DESCRIPTION
**Pull request by rudaporto**
This fixes #32 

Since tox do not support python3.3 I've also removed it from travis and tox configuration.

### CODE REVIEW

File: logzio/sender.py [hunk 1]
```diff
@@ -153,12 +158,19 @@ def _flush_queue(self):
...
         current_size = 0
         while not self.queue.empty():
             current_log = self.queue.get()
-            current_size += sys.getsizeof(current_log)
+            try:
+                current_size += sys.getsizeof(current_log)
+            except TypeError:
+                # pypy do not support sys.getsizeof
+                current_size += len(current_log) * 4
+
```
Comment (idohalevi) [inline logzio/sender.py]: @roiravhon what do you think?
I think the upper representation is 4 so in the worst case we will send a smaller bulks

Comment (roiravhon) [reply logzio/sender.py]: @idohalevi we can go all philosophical on that manner, but I think this should work.
Can we just verify with large data set that the bulks are still in the right size?

**Comment by idohalevi** [comment]
@rudaporto 
Looks good. 
I'll merge and publish next week

**Comment by rudaporto** [comment]
> @rudaporto
> Looks good.
> I'll merge and publish next week

Thanks, @idohalevi

**Pull request merged**